snuba/settings.py

```diff
@@ -13,6 +13,8 @@
 DEFAULT_DATASET_NAME = "events"
 DISABLED_DATASETS: Set[str] = set()
 
+UPDATE_TRANSACTION_NAME = False
```

Comment:
Why is is possible for users to define this value? I don't think the bare `unqualified_query_view` is hugely useful for anyone, it seems to me we could just apply the new name

Reply:
Is this something that would change for self serve etc.? I just didn't want to mess up other people's setups by changing this unilaterally.

Reply:
I guess it could possibly change for self serve if they are using another Sentry instance to trace their Sentry. I find it hard to imagine why anyone would be relying on this value always being `unqualified_query_view` as part of their setup though. In any case, I don't think our Sentry transaction names (or most other error or transaction metadata sent to Sentry) should ever be configurable. If we rename this function or refactor this code, it would also naturally change. I don't think we should add configuration that attempts to preserve a particular value.

Reply:
OK, I'll remove this. I'm fine with that.